languages/elixir/concepts/cond/links.json

```diff
@@ -1 +1,10 @@
-[]
+[
+  {
+    "url": "https://elixir-lang.org/getting-started/case-cond-and-if.html#cond",
+    "description": "cond"
+  },
+  {
+    "url": "https://elixir-lang.org/getting-started/case-cond-and-if.html#case",
+    "description": "case"
+  }
+]
```

Comment:
@jwarwick Let's fix this file too, before merging

Reply:
Good catch. Will update.